category_theory/limits/limits_from_equalizers_and_products.lean

```diff
@@ -0,0 +1,54 @@
+-- Copyright (c) 2017 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Scott Morrison
+
+import category_theory.limits
+
+open category_theory
+
+namespace category_theory.limits
+
+universes u₁ v₁
+variables {C : Type u₁} [category.{u₁ v₁} C]
+
+def limits_from_equalizers_and_products
+  [has_products.{u₁ v₁} C] [has_equalizers.{u₁ v₁} C] : has_limits.{u₁ v₁} C :=
+{ cone := λ J 𝒥 F,
+  begin
```

Comment:
tactic in definition?

Reply:
`resetI`, `let` and `exact` is okay. (mostly avoiding tactics in definitions is to avoiding spurious rewrites)